ladybug/wea.py

```diff
@@ -78,16 +78,23 @@ def from_json(cls, data):
                    diffuse_horizontal_radiation, timestep)
 
     @classmethod
-    def from_epw_file(cls, epwfile):
+    def from_epw_file(cls, epwfile, timestep=1):
         """Create a wea object using the solar radiation values in an epw file.
 
         Args:
             epwfile: Full path to epw weather file.
+            timestep: An optional integer to set the number of time steps per hour.
+                Default is 1 for one value per hour.
         """
         epw = EPW(epwfile)
-        return cls(epw.location,
-                   epw.direct_normal_radiation,
-                   epw.diffuse_horizontal_radiation)
+        direct_normal = epw.direct_normal_radiation
+        diffuse_horizontal = epw.diffuse_horizontal_radiation
+        if timestep is not 1:
+            direct_normal = direct_normal.interpolate_data(timestep, True)
```

Comment:
I think we should add a warning here to let the user now current interpolation is linear and good for energy modeling however is not appropriate for daylight studies.

Reply:
Integrated a warning into the docsting.  Let me know if it belongs anywhere else.

Reply:
Looks good! This will be great for developers who will see the source code but I think we should also make it explicit and print out the warning for users if timestep is not 1 and the wea is created from an epw file.

Reply:
Understood.  Yes, when I expose it on the component, I will add a warning.

Reply:
... or do you mean that we should be printing a warning?